assets/css/case-studies.scss

```
@@ -2230,3 +2230,133 @@ $csi-line: #e7edec; // hairline on light cards
+      overflow: visible !important;
+    }
+    .cs-hero__inner--framed {
+      backdrop-filter: none !important;
+      -webkit-backdrop-filter: none !important;
+    }
+
+    // Avoid slicing self-contained blocks across page breaks. A page break that
+    // lands mid-card chops off its top/bottom edge (the clipping seen on the
+    // Business Impact cards). `break-inside: avoid` pushes the whole block to the
+    // next page instead of splitting it.
+    //
```

Stylelint (17.13.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)